group-actors/src/main/resources/activityConfig.json

```diff
@@ -0,0 +1,23 @@
+{
+  "activities":[
+    {
+      "type": [
+        "Course"
+      ],
+      "baseUrl": "CONTENT_SERVICE_PORT",
+      "apiUrl": "sunbird_cs_search_url",
+      "request": {
+              "id":"identifier",
```

Comment:
@sknirmalkar89 - not sure if this request structure is correct for content search API. The request body for search API should have a 'filters' section.